monai/apps/manifest/reference_resolver.py

```diff
@@ -65,11 +68,11 @@ def add_item(self, item: ConfigItem):
         """
         id = item.get_id()
         if id in self.items:
-            warnings.warn(f"id '{id}' is already added.")
+            # warnings.warn(f"id '{id}' is already added.")
```

Comment:
mute this warning because we may repeatedly parse configs

Reply:
OK, I removed this comment directly.

Thanks.